worker_test.go

```diff
@@ -452,3 +511,23 @@ func cleanKeyspace(namespace string, pool *redis.Pool) {
 		}
 	}
 }
+
+func pauseJobs(namespace, jobName string, pool *redis.Pool) {
```

Comment:
It will be nice for test helpers to return the error, because the caller can then decide on the control flow instead of always abort. If you still prefer to abort, `t.Fatalf` is prefered.